.github/workflows/test.yml

```
@@ -71,6 +71,8 @@ jobs:
             voicevox_engine_asset_name: linux-cpu
           - os: macos-latest
             voicevox_engine_asset_name: macos-x64
+          - os: macos-latest
+            voicevox_engine_asset_name: macos-arm64
```

テストするのはx64だけでOKだと思います！今は。

`voicevox_nemo_negine`がarm64に対応したら変更する、というメモを残すのはどうでしょう？
```suggestion
          - os: macos-latest
            voicevox_engine_asset_name: macos-x64
            # voicevox_nemo_negineがarm64に対応したら変更する
            # voicevox_engine_asset_name: macos-arm64
```